alpha-lab/phone-neon/index.md

```diff
@@ -1,118 +1,103 @@
+    content: "https://www.youtube.com/embed/gp5O1uskDME"
+  - name: twitter:width
+    content: "1280"
+  - name: twitter:height
+    content: "720"
+  - property: og:image
+    content: "https://i.ytimg.com/vi/gp5O1uskDME/maxresdefault.jpg"
 tags: [Neon, Cloud]
 ---
 
 <script setup>
 import TagLinks from '@components/TagLinks.vue'
 </script>
 
-# Uncover gaze behaviour on phone screens with Neon
+# Uncover Gaze Behaviour on Phone Screens With Neon
 
 <TagLinks :tags="$frontmatter.tags" />
 
 <Youtube src="gp5O1uskDME"/>
 
-Have you ever wondered what your eyes focus on when scrolling through your favourite app?
+What catches your attention and how do you visually navigate through the interface/page of a mobile app? This article shows you the tools you can use to address these questions. 
 
-In this article, we test whether Neon, our latest eye tracker, can accurately capture and characterise viewing behaviour
-while gazing at small icons on mobile applications. To achieve this, we used some of Alpha Lab's existing tutorials, including
-how to generate scanpaths, define areas of interest (AOIs) and calculate outcome metrics, and map gaze onto dynamic content.
+## Gaze Behaviour During Mobile App Engagement
 
-Shall we start?
+Researchers in fields such as UI/UX and neuromarketing frequently investigate user visual engagement with mobile applications. Wearable eye tracking is sometimes used for this purpose as it can provide valuable insights into user experience during mobile browsing. In this article, we used our latest wearable eye tracker, Neon, to capture and analyse users' viewing behaviour when focusing on small icons and features of mobile applications.
 
-## What you'll need
+## Overcoming Usability Challenges With Neon
 
-Below you can find the tools we used for this project. Using these, you can replicate the content of this article with
-your own applications.
+In the past, wearable eye tracking faced challenges in usability primarily due to calibration limitations, which may have made mobile browsing measurements difficult and/or time-consuming in practice. However, we believe that Neon overcomes these obstacles by offering calibration-free gaze estimation with very good accuracy. For this reason, we wanted to try out Neon when tracking gaze during mobile browsing, and put together an article so you can replicate what we did.
 
-### Cloud enrichment
+## Leveraging Pupil Cloud and Alpha Lab Tutorials To Explore User Gaze Dynamics
 
-- [Reference Image Mapper](https://docs.pupil-labs.com/neon/pupil-cloud/enrichments/reference-image-mapper/)
+In order to characterise viewing behaviour on mobile phones using Neon, we first needed to make some eye tracking recordings whilst a user was engaging with a phone. We then leveraged Pupil Cloud and Alpha Lab tutorials to interpret the data and compute further metrics.
 
-### Alpha Lab tutorials
+Firstly, we placed the phone in a static position on a desk, allowing us to interact with the app.
```

Comment:
Why does the phone need to be static? What is the motivation/constraint here? Reader needs to know that this is done so we can easily compare gaze data across recordings using RIM.

Reply:
Thanks for the feedback. I'll add an explanation: "1. We placed the phone in a static position on a desk, allowing us to interact with the app. This setup was crucial for utilizing our [Reference Image Mapper](https://docs.pupil-labs.com/neon/pupil-cloud/enrichments/reference-image-mapper/) enrichment which specifically requires the scene to have relatively static features in the environment."

Reply:
Could also enable comparison across subjects right?